src/pages/Pool/components/LocksList/LockListView.tsx

```diff
@@ -10,7 +11,13 @@ interface LockListViewProps {
 
 export const LockListView: FC<LockListViewProps> = ({ positions }) => {
   return (
-    <List dataSource={positions} gap={2}>
+    <List
+      dataSource={positions}
+      gap={2}
+      emptyTemplate={
+        <EmptyTemplateContainer>No results found</EmptyTemplateContainer>
```

Comment:
Can you move it to a separate components. Used two times.
It may cause a mistake when we will launch i18n